internal/jimm/ssh/ssh.go

```diff
@@ -51,11 +54,11 @@ type JWTGeneratorFactory interface {
 }
 
 // NewSSHManager returns a new SSHManager that offers jimm functionality to the SSHJumpServer.
-func NewSSHManager(identityManager IdentityManager, modelManager ModelManager, sshKeyManager SSHKeyManager, jwtFactory JWTGeneratorFactory) (*sshManager, error) {
+func NewSSHManager(identityManager IdentityManager, jujuManager JujuManager, sshKeyManager SSHKeyManager, jwtFactory JWTGeneratorFactory) (*sshManager, error) {
```

Comment:
Following what Juju does, we could have a pattern of params struct with a .Validate() method, I kinda like it in Juju.